### PR TITLE
update: TürkAnimeTV presence

### PR DESCRIPTION
**Pull request by emirkabal**
* Added presence buttons
* Added a few strings
* Added a few pages
* Fix episode matching

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by emirkabal** [comment]
> Upload proof of it working

File: websites/T/TürkAnimeTV/presence.ts [hunk 1]
```diff
@@ -70,7 +78,7 @@ presence.on("UpdateData", async () => {
       : (await strings).playing;
 
     if (!video.paused && video.duration) {
-      const timestamps = getTimestamps(
+      const timestamps = presence.getTimestamps(
```
Comment (ririxidev) [inline websites/T/TürkAnimeTV/presence.ts:81]: use array deconstructing

**Review by rxri — CHANGES_REQUESTED** [review]
(fix eslint issues)

File: websites/T/TürkAnimeTV/presence.ts (hunk 1 above)
Comment (emirkabal) [reply websites/T/TürkAnimeTV/presence.ts:81]: oh sorry

File: websites/T/TürkAnimeTV/presence.ts [hunk 2]
```diff
@@ -70,12 +73,12 @@ presence.on("UpdateData", async () => {
...
 
     if (!video.paused && video.duration) {
-      const timestamps = getTimestamps(
+      const [currentTime, duration] = presence.getTimestamps(
         Math.floor(video.currentTime),
         Math.floor(video.duration)
       );
-      presenceData.startTimestamp = timestamps[0];
-      presenceData.endTimestamp = timestamps[1];
+      presenceData.startTimestamp = currentTime;
```
Comment (ririxidev) [inline websites/T/TürkAnimeTV/presence.ts:80]:
```suggestion
```

File: websites/T/TürkAnimeTV/presence.ts [hunk 3]
```diff
@@ -70,12 +73,12 @@ presence.on("UpdateData", async () => {
       : (await strings).playing;
 
     if (!video.paused && video.duration) {
-      const timestamps = getTimestamps(
+      const [currentTime, duration] = presence.getTimestamps(
```
Comment (ririxidev) [inline websites/T/TürkAnimeTV/presence.ts:76]:
```suggestion
      const [, duration] = presence.getTimestamps(
```

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**